docs/products/clickhouse/getting-started.rst

```diff
@@ -0,0 +1,45 @@
+Getting started
+===============
+
+The first step in using Aiven for ClickHouse is to create a service. You can do this in the `Aiven web console <https://console.aiven.io/>`_ or with the `Aiven CLI <https://github.com/aiven/aiven-client>`_.
+
+
+Create a ClickHouse service in the Aiven web console
+----------------------------------------------------
+
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_.
+
+2. On the *Services* page, click **Create a new service**.
+
+3. Select the main properties for your service:
+
+   a. Select **ClickHouse** as the service type.
+
+   b. Select the cloud provider and region that you want to run your service on.
+
+      .. note:: 
+	      The pricing for the same service may vary between different providers and regions. The service summary on the right side of the console shows you the pricing for your selected options.
+
+   c. Select a service plan.
+
+      This defines the number of nodes, storage, memory, CPU, and backup support for the service.
+
+   d. Enter a name for your service.
```

Comment:
The name cannot be changed later, should we specify this here? or a warning in the ui is enough?

Reply:
I think it's enough that the UI includes a warning about this - I think we don't mention it in the documentation for the other services either.